nemo_skills/pipeline/summarize_results.py

```diff
@@ -109,6 +109,9 @@ def summarize_results(
     eval_results_dir = Path(results_dir) / 'eval-results'
     if eval_results_dir.exists() and eval_results_dir.is_dir():
         results_dir = eval_results_dir
+    else:
+        raise ValueError(f"No 'eval-results' dir found in {results_dir}")
```

Comment:
that's not the right way to do it I think. The else part might normally happen if you provide a path to eval-results itself, so we shouldn't throw an error here. Instead what we should check is whether the dirname corresponds to one of the benchmarks we know how to compute metrics for and whether we have any output files there

Reply:
Ok, so there are three options possible.

1. `{results_dir}/eval-results/{benchmark}/output*jsonl`
2. `{results_dir}/{benchmark}/output*jsonl`
3. `{results_dir}/output*jsonl`

So we can support all three options in this PR. Is that what you're recommending above?

Reply:
Yes, all 3 options are possible. The first two should check if benchmark is something we have metrics for (unless metrics_type is specified), and the third option should check if metrics type is explicitly provided. And in all cases, we should have a clear error if there are no output*.jsonl inside any of the folders